bindings/opencv/dnn/main.cpp

```diff
@@ -122,48 +123,56 @@ int main(int argc, char *argv[]) {
         p[i] = cv::saturate_cast<uchar>(pow(i / 255.0, gamma) * 255.0);
 
     while (cv::waitKey(1) != 27 &&
-           getWindowProperty("Display Image", cv::WND_PROP_AUTOSIZE) >= 0) {
+           getWindowProperty("Display Objects", cv::WND_PROP_AUTOSIZE) >= 0) {
         /* Request frame from camera */
         status = camera->requestFrame(&frame);
         if (status != Status::OK) {
             LOG(ERROR) << "Could not request frame!";
             return 0;
         }
 
-        /* Convert from frame to depth mat */
+        /* Obtain the depth mat from the frame, this will be used for distance
+         * calculation*/
         cv::Mat frameMat;
         status = fromFrameToDepthMat(frame, frameMat);
         if (status != Status::OK) {
             LOG(ERROR) << "Could not convert from frame to mat!";
             return 0;
         }
 
-        /* Obtain the depth mat from the frame in order to determine
-         * the distance of the identified objects */
+        /* Obtain the depth mat from the frame */
         cv::Mat depthMat;
         status = fromFrameToDepthMat(frame, depthMat);
         if (status != Status::OK) {
             LOG(ERROR) << "Could not convert from frame to mat!";
             return 0;
         }
 
+        /* Obtain the ir mat from the frame */
+        cv::Mat irMat;
+        status = fromFrameToIrMat(frame, irMat);
+        if (status != Status::OK) {
+            LOG(ERROR) << "Could not convert from frame to mat!";
+            return 0;
```

Comment:
The 0 return value is associated with success. You could return something different from 0, like -1.

Reply:
modified in commit: 12ef2f1c34a3c0a54f11b489560750b1f119899f